geointel/cli_formatter.py

```
@@ -0,0 +1,92 @@
+def format_results(results: Dict[str, Any]) -> str:
+    """
+    Format successful results for display.
+    
+    Args:
+        results: Dictionary containing the analysis results
+        
+    Returns:
+        Formatted results string with ANSI color codes
+    """
+    output = "\n\033[92m===== Analysis Results =====\033[0m\n"
+    output += f"\033[96mInterpretation:\033[0m\n"
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)